UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/NoticestoMarinersController.cs

```diff
@@ -80,13 +80,22 @@ public async Task<IActionResult> ShowWeeklyFilesAsync(int year, int week)
 
         public async Task<IActionResult> ShowDailyFilesAsync()
         {
```

Comment:
There one more action method 'public IActionResult DailyFiles()' for Daily NM files, is this used for getting data? if not please remove it.

Reply:
removed DailyFiles fuction